Admin.java

```diff
@@ -0,0 +1,113 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class Admin {
+    private Scanner scanner = new Scanner(System.in);
+    private ArrayList<HashMap<String, Object>> spaces = new ArrayList<>();
```

Comment:
why we need hashmap inside list? its over engineering, you can simply use ArrayList and directly store objects )

Reply:
Thanks for the review. I have added Reservations and Spaces classes to store the data.